app/src/main/java/org/mozilla/fenix/settings/address/AddressEditorFragment.kt

```diff
@@ -41,18 +54,44 @@ class AddressEditorFragment : SecureFragment(R.layout.fragment_address_editor) {
         )
 
         val binding = FragmentAddressEditorBinding.bind(view)
+        setHasOptionsMenu(true)
 
-        addressEditorView = AddressEditorView(binding, interactor)
+        addressEditorView = AddressEditorView(binding, interactor, args.address)
```

Comment:
Maybe we should pass the address through the `bind()` instead? This seems cleaner to me instead of having a nullable property in `AddressEditorView`. Also, it just follows the current CreditCardEditorView convention.

Reply:
I would prefer to have the `address` as property because we also use it in other places (the `saveAddress` method) and I would prefer not to pass it around as param from method to method.

Reply:
Ok, that makes sense to me.